### PR TITLE
Simplify the contribution guide

### PR DESCRIPTION
**Pull request by jmg-duarte**
This relates to #348, there are other documents on the list to be deleted or merged in the future as this is a work in progress

### CODE REVIEW

**Comment by riccardoferretti** [comment]
@jmg-duarte let me know when the PR is ready for review/merge, I will wait for the pieces to be all there before leaving comments

**Comment by riccardoferretti** [comment]
I was thinking it could be good to add 2 things:
- a section on "setting up your environment"
- an action item to "consider updating this guide, in case you found anything out of date or to be improved, your first contribution :)"

wdyt?

**Comment by jmg-duarte** [comment]
This is ready for review

File: docs/contribution-guide.md [hunk 1]
```diff
@@ -29,25 +32,36 @@ If you're interested in contributing to the VS Code extension (aka `foam-vscode`
 
    `yarn install`
 
-3. This project uses [Yarn workspaces](https://classic.yarnpkg.com/en/docs/workspaces/).`foam-vscode` relies on `foam-core`. This means we need to compile it before we do any extension development. From the root, run the command:
+3. This project uses [Yarn workspaces](https://classic.yarnpkg.com/en/docs/workspaces/). `foam-vscode` relies on `foam-core`. This means we need to compile it before we do any extension development. From the root, run the command:
+
+   `yarn workspace build`
```
Comment (riccardoferretti) [inline docs/contribution-guide.md:37]: minor, I simply use `yarn build`
```suggestion
   `yarn build`
```

File: docs/contribution-guide.md [hunk 2]
```diff
@@ -29,25 +32,36 @@ If you're interested in contributing to the VS Code extension (aka `foam-vscode`
...
+You should now be ready to start working!
+
+### The VS Code Extension
+
+This guide assumes you read the previous instructions and you're set up to work on Foam.
+
+1. Now we'll use the launch configuration defined at [`.vscode/launch.json`](https://github.com/foambubble/foam/blob/master/.vscode/launch.json) to start a new extension host of VS Code. From the root, or the `foam-vscode` workspace, press f5.
 
-   `yarn workspace foam-core build`
+2. In the new extension host of VS Code that launched, open a Foam workspace (e.g. your personal one, or a test-specific one created from foam-template). This is strictly not necessary, but the extension won't auto-run unless it's in a workspace with a `.vscode/foam.json` file.
```
Comment (riccardoferretti) [inline docs/contribution-guide.md:47]: minor, I would make `foam-template` a link

```suggestion
2. In the new extension host of VS Code that launched, open a Foam workspace (e.g. your personal one, or a test-specific one created from [foam-template](https://github.com/foambubble/foam-template)). This is strictly not necessary, but the extension won't auto-run unless it's in a workspace with a `.vscode/foam.json` file.
```

**Review by riccardoferretti — APPROVED** [review]
only a couple of minor comments, PR looks good, feel free to merge

File: docs/contribution-guide.md (hunk 1 above)
Comment (jmg-duarte) [reply docs/contribution-guide.md:37]: Same tbh, I just kept the old one. I'll replace

**Comment by jmg-duarte** [comment]
Suggestions added, merging and closing

**Comment by riccardoferretti** [comment]
@all-contributors add @jmg-duarte for docs

**Pull request merged**